css/style.css

```diff
@@ -0,0 +1,485 @@
+@import url('./fonts.css');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.level, .play {
+  cursor: pointer;
+}
+
+.play {
+  margin-top: 88px;
+  min-width: 352px;
+  min-height: 95px;
+  background: #D62B2E;
+  border-radius: 7px;
+  font-size: 32px;
+  color: #FFFFFF;
+}
+
+.play:hover {
+  background: blue;
+}
+
+body {
+  background-image: url(../img/Background.png);
+  background-size: cover;
+  font-family: 'Roboto Bold', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+/* .container {
+  max-width: 70%;
+  margin: auto;
+} */
+
+.game {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  align-items: center;
+}
+
+.menu {
+  width: 85%;
+  position: relative;
+}
+
+.level-choice__header {
+  margin-bottom: 16px;
+  font-size: 54px;
+  color: #FFFFFF;
+}
+
+.level-choice__instruction {
+  display: block;
+  margin-bottom: 60px;
+  font-size: 18px;
+  color: #FFFFFF;
+}
+
+.level-choice__wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-content: center;
+  align-items: start;
+  max-height: 300px;
+}
+
+.level {
+  position: relative;
+  display: block;
+  margin-bottom: 24px;
+  font-size: 48px;
+  line-height: 59px;
+  color: #FFFFFF;
+  margin-left: 46px;
+}
+
+.lvl-active {
+  position: relative;
+}
+
+.lvl-active::after {
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #152fa1;
+  transform: rotate(45deg);
+}
+
+.level::before {
+  display: none;
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #D62B2E;
+  transform: rotate(45deg);
+}
+
+.level:hover::before {
+  display: block;
+}
+
+.first-card {
+  position: absolute;
+  left: 60%;
+  top: 35%;
+  transform: rotate(11.9deg);
+  z-index: -2;
+}
+
+.second-card {
+  position: absolute;
+  left: 69%;
+  top: 45%;
+  transform: rotate(39.25deg);
+  z-index: -1;
+}
+
+.hide {
+  display: none;
+}
+
+.table {
+  display: none;
+  /* width: 700px; */
+  width: 100%;
+  /* align-items: start; */
+}
+
+.table-0 {
+  width: 720px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.table-1 {
+  display: flex;
+  height: 94%;
+  max-height: 720px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.table-2 {
+  display: flex;
+  justify-content: center;
+  width: 600px;
+  height: 95%;
+  flex-wrap: wrap;
+}
+
+
+
+
+.card-wrapper {
+  outline: 2px solid blue;
+  /* width: 31%; */
+  width: 218px;
+  height: 341px;
+  text-align: center;
+  perspective: 1200;
+  -webkit-perspective: 1200;
+  -moz-transform: perspective(1200px);
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)